Dockerfile.c12e-ci.cortex-python-lib

```diff
@@ -1,4 +1,4 @@
-FROM python:3.10.2-slim
+FROM python:3.10.5-slim
```

Comment:
We've had some issues with this in the past (mainly for the nodejs builds) where a newer patch version of the base image would break certain things :( 

IMO we can use the more generic tag until we hit an issue with the pipeline not behaving right before we start using the idempotent tag.